Assets/Tests/EditMode/AuthTests/RestClientRequestAdapterTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using NUnit.Framework;
+using Src.AuthController.REST;
+
+namespace Tests.EditMode.AuthTests
+{
+    public class RestClientRequestAdapterTests
+    {
+        [Test]
+        [TestCaseSource(nameof(GetRequestMethodTypes))]
+        public void GetRequestMethodName_ShouldReturnValidName(RestRequestMethodType type)
```

Comment:
The fact that method does not throw any exception is only part of expected behaviour. More tests should be added.